src/common/components/badge/index.js

```diff
@@ -7,14 +7,14 @@ import { CopyToClipboard } from '../share';
 
 import style from './badge.css';
 
-const getBadgeUrl = (fullName) => `${BASE_URL}/badge/${fullName}.svg`;
+const getBadgeUrl = (fullName, buildStatus) => `${BASE_URL}/badge/${fullName}.svg${buildStatus ? '?'+encodeURIComponent(buildStatus) : ''}`;
```

Comment:
Passing current build status to badge URL to force browser to reload the image when build status changes.